reviews/2022-metal3-io-annual.md

```diff
@@ -0,0 +1,264 @@
+  - Bi-weekly triaging of issues.
+  - Restrict interface usage of ironic for better security.
+  - Rename all the ‘master’ branches to ‘main’.
+
+## Future Goals
+
+  - One of the main goals for 2022 was to help adoption of Metal3 thus the
+    Metal3 community has started writing a [User-guide](https://metal3io.netlify.app).
+    The community plans to extend the guide continuously and also provide
+    separate tutorials that will showcase the deployment process on bare metal
+    environment that is as close to a production environment as possible.
+
+  - In 2022 certain aspects of Metal3 security has been improved, but there
+    is more planned specially related to the topics of container security
+    and related to the testing of Metal3 security features during e2e tests.
+
+  - There are improvements done on the CI infrastructure of the Metal3
+    project, but there are more to come in terms of securing and speeding the
+    test processes to help contributors receive feedback faster.
+
+  - There is an ongoing process that will continue in the future to align
+    the Metal3 e2e test better with the CAPI e2e tests as well as to rework
+    the developer environment that is currently used to better fit the CI
+    use cases.
+
+  - There are ongoing discussion related to expanding the features of BMO,
+    CAPM3 and IPAM e.g.:
+    - Adding better support for Ironic root device hint operators.
+    - Supporting use-cases where the user would like to manage the lifecycle
+      of bare metal machines from a K8S cluster but don't want to include
+      the machines as nodes in the cluster.
+    - Adding support for the CAPI IPAM contract.
+    - Adding support for selective disk cleaning in baremetal-operator.
+    - Adding support for multitenancy.
+
+## How can the CNCF help you achieve your future goals?
+
+1. We would like to ask CNCF to guide us through the fuzzing process using [CNCF fuzzing](https://github.com/cncf/cncf-fuzzing).
+
+2. We would like to ask CNCF to help Metal3-io project to improve its visibility
+   in any way possible e.g.:
+     - finding advocate(s) for the project
+     - facilitating cooperation between Metal3 and other FOSS projects
+     - or providing with Metal3-io with any other opportunity to improve its
+       traction/visibility
+
+
+## Security processes
+
+The Metal3-io project's general security policy is located [here](https://metal3io.netlify.app/security_policy.html).
+
+In addition to the project's security policy, all the active repositories
+have a SECURITY_CONTACTS file. The SECURITY_CONTACTS file informs
+contributors about:
+  - the project's security policy
+  - basic rules of security vulnerability disclosure
+
+Optionally the SECURITY_CONTACTS may contain:
+  - repository specific security fix back porting information
+  - repository specific security fix release information
+
+## Incubation
+
+We think we meet the criteria of incubation. We have shown healthy number of
```

Comment:
We think Metal3 (or Metal3-io) meets ...

Reply:
Thanks, done.